homeassistant/components/evohome/__init__.py

```diff
@@ -270,29 +271,29 @@ def __init__(self, evo_broker, evo_device) -> None:
         self._state_attributes = []
 
         self._supported_features = None
-        self._setpoints = None
+        self._schedule = {}
 
     @callback
     def _refresh(self, packet):
         if packet['signal'] == 'refresh':
             self.async_schedule_update_ha_state(force_refresh=True)
 
-    def get_setpoints(self) -> Optional[Dict[str, Any]]:
-        """Return the current/next scheduled switchpoints.
+    @property
+    def setpoints(self) -> Optional[Dict[str, Any]]:
```

Comment:
Optional is wrong

Reply:
A empty dict make it not optimal for typing, only if you return None or a value

Reply:
Yes - will fix it.